src/abi/erc165.json

```diff
@@ -0,0 +1,21 @@
+[
```

Comment:
Watch the indentation here! Check out erc20.json.

Reply:
what plugin did you use to indentation?